ui/ducks/send/send.js

```diff
@@ -2679,7 +2679,7 @@ export function updateSendAsset(
 
       if (details.standard === TokenStandard.ERC20) {
         asset.balance =
-          details.balance && details.decimals
+          details.balance && details.decimals >= 0
```

Comment:
```suggestion
          details.balance && typeof details.decimals === 'number'
```
I guess we're actually interested in checking if it's properly defined as a `number` at all? This avoid typecasting as part of comparison.

Reply:
Should it actually handle and forward 0 balance as well, to properly distinguish between the cases `balance is known to be 0` and `balance undefined or couldn't be determined`? 
```suggestion
          typeof details.balance === 'string' && typeof details.decimals === 'number'
```